data/helpers/dbHelpers.js

```diff
@@ -0,0 +1,17 @@
+const db = require('../dbConfig.js');
+
+module.exports = {
+  getAllUsers: () => {
+    return db('users');
+  },
+  findByUsername: (username) => {
+    return db('users').where('username', username);
+  },
+  addUser: (user) => {
+    return db('users').insert(user);
+  },
+  settings: {
```

Comment:
I like the settings object you added, I think it's in a good place too as it helps.